pages/studentPortal/quizcomponent/index.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from "react";
+
+type SkillSelectionProps = {
+  selections: string[];
+};
+
+const SkillSelection: React.FC<SkillSelectionProps> = ({ selections }) => {
+  const [selected, setSelected] = useState<string[]>([]);
+
+  const handleSelection = (selection: string) => {
+    if (selected.includes(selection)) {
+      setSelected(selected.filter((s) => s !== selection));
```

Comment:
nice use of filter and spread operator to support adding and removing from the list. I think it's good for now, but we may want to move this into redux later once we get more components. Perhaps redux will be overkill, so we can keep this for now and see how the feature progresses. Just something you should think about